ds/beis_indicators/patents/make_patents.py

```diff
@@ -0,0 +1,178 @@
+#Script to make HESA indicators
+
+import csv
+import zipfile
+import io
+import os
+import requests
+import beis_indicators
+import numpy as np
+import json
+from data_getters.labs.core import download_file
+import random
+from ast import literal_eval
+
+from beis_indicators.utils.dir_file_management import *
```

Comment:
minor, I'd sort these alphabetically, so that we can see that we have multiple import statements related to `beis_indicators`